doc/docs/master/how-tos/advanced-data-operations/use-transactions-to-run-multiple-commands.md

```diff
@@ -4,19 +4,164 @@
+in the pipeline repo. After the transaction, you
+can put files and finish the commits at will, and **the pipeline job
+will run once all the input commits have been finished**.
 
+See transactions as a Flow Control feature.
+
+## Start and Finish Transaction Demarcations
+
+
+!!! Important "Preamble"
+    A transaction demarcation initializes some transactional behavior before the demarcated area begins, then ends that transactional behavior when the demarcated area ends. You should see those demarcations as a declaration of the group of commands that will be treated together as a single coherent operation.
+
+
+* To start a transaction demarcation, run the following command:
+
+    ```shell
+    pachctl start transaction
+    ```
+
+    **System Response:**
+
+    ```shell
+    Started new transaction: 7a81eab5e6c6430aa5c01deb06852ca5
+    ```
+
+    This command generates a transaction object in the cluster and saves
+    its ID in the local Pachyderm configuration file. By default, this file
+    is stored at `~/.pachyderm/config.json`.
+
+    !!! example
+          ```json hl_lines="9"
+          {
+            "user_id": "b4fe4317-be21-4836-824f-6661c68b8fba",
+            "v2": {
+              "active_context": "local-2",
+              "contexts": {
+                "default": {},
+                "local-2": {
+                  "source": 3,
+                  "active_transaction": "7a81eab5e6c6430aa5c01deb06852ca5",
+                  "cluster_name": "minikube",
+                  "auth_info": "minikube",
+                  "namespace": "default"
+                },
+          ```
+
+  
+    After you start a transaction demarcation, you can add [supported commands (i.e., transactional commands)](#supported-operations), such
+    as `pachctl start commit`, `pachctl create branch` ..., to the
+    transaction.  
+
+    All commands that are performed in a transaction are
+    queued up and not executed against the actual cluster until you finish
+    the transaction. When you finish the transaction, all queued command
+    are executed atomically.
+
+* To finish a transaction, run:
+
+    ```shell
+    pachctl finish transaction
+    ```
+
+    **System Response:**
+
+    ```shell
+    Completed transaction with 1 requests: 7a81eab5e6c6430aa5c01deb06852ca5
+    ```
+
+    !!! tip "Noteworthy"
+          As soon as a commit is started (whether through `start commit` or `put file` without an open commit, or finishing a transaction that contains a start commit), a new [**commitset** as well as a **jobset**](../../../concepts/advanced-concepts/globalID/#definition) is created. All open commits are in a `started` state, each of the pipeline jobs created is `running`, and the worker responsible for dividing up the tasks is waiting for the commit(s) to be closed to process the data. In other words, your changes will only be applied when you close the commits.
```

Comment:
it's probably fine to just say "workers" rather than "the worker responsible for dividing up the tasks" - I know those were my words, but that's more a normal detail of how jobs work